### ISSUE TITLE
Kernel fails to start

### ISSUE THREAD

**Issue by siclait** [clojupyter/clojupyter#112]
I built a kernel according to the instructions in https://github.com/clojupyter/clojupyter/blob/master/doc/library.md but when I attempt to start Jupyter Lab with the kernel, I get the following error:

```
Error: Could not find or load main class clojupyter.kernel.core
Caused by: java.lang.ClassNotFoundException: clojupyter.kernel.core
```

When I take a look at the `kernel.json`, I see the following:
```json
{
  "argv": ["java", "-cp", "/path/to/clojupyter-standalone.jar", "clojupyter.kernel.core", "{connection_file}"],
  "display_name": "Clojure (clojure)",
  "language": "clojure",
  "interrupt_mode": "message"
}
```

If I modify the `kernel.json` to call `clojure.main` instead of `clojupyter.kernel.core`, everything works:
```json
{
  "argv": ["java", "-cp", "/path/to/clojupyter-standalone.jar", "clojure.main", "-m", "clojupyter.kernel.core", "{connection_file}"],
```

Am I doing something wrong?

Here is my `deps.edn`:
```edn
{:deps    {clojupyter               {:mvn/version "0.3.1"}
           org.clojure/clojure      {:mvn/version "1.10.1"}}
 :aliases {:depstar
           {:extra-deps
            {seancorfield/depstar {:mvn/version "0.5.2"}}}}}
```

**Comment by klausharbo** [comment]
Your fix looks fine, at least it works for me too.  And I don't see that you are doing anything wrong - this should work, and used to work.  It looks like I accidently dropped the AOT compilation of `clojupyter.cmdline` in the transition to using `deps.edn` for dependencies.   I will try to release a fix soon.

Sorry for responding so late - I must somehow have missed the notification I otherwise receive when Clojupyter issues are posted.

-Klaus.

**Comment by klausharbo** [comment]
I just released v0.3.2 which I believe addresses this issue, i.e. it should be possible to follow the documentation mentioned above a get a working result.  The only difference between 0.3.1 and 0.3.2 is AOT add directives in `project.clj`:

```
  :main				clojupyter.kernel.core
  :aot				[clojupyter.cmdline]
```

**Comment by siclait** [comment]
Thank you, this works perfectly.
